fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMaker.java

```diff
@@ -57,33 +81,29 @@ public ArithmeticMaker(
      * @param function  The arithmetic operation performed by the LogicalMetrics constructed by this maker
      */
     public ArithmeticMaker(MetricDictionary metricDictionary, ArithmeticPostAggregationFunction function) {
-        // TODO: Deprecate me, mappers should always be specified at creation time, not implicitly
-        this(metricDictionary, function, new SketchRoundUpMapper());
+        this(
+                metricDictionary,
+                function,
+                (Function<String, ResultSetMapper>) (ignore) -> NO_OP_MAPPER
```

Comment:
Parens around `ignore` (ie. the lambda input variable) are not technically needed, since there's no type on it and there's only 1 variable.

/NotABlocker